metaflow/cli.py

```diff
@@ -937,14 +937,17 @@ def start(
     ctx.obj.package_suffixes = package_suffixes.split(",")
     ctx.obj.reconstruct_cli = _reconstruct_cli
 
-    ctx.obj.event_logger = EventLogger(event_logger)
-
     ctx.obj.environment = [
         e for e in ENVIRONMENTS + [MetaflowEnvironment] if e.TYPE == environment
     ][0](ctx.obj.flow)
     ctx.obj.environment.validate_environment(echo)
 
-    ctx.obj.monitor = Monitor(monitor, ctx.obj.environment, ctx.obj.flow.name)
+    ctx.obj.event_logger = EventLogger(
+        event_logger, ctx.obj.environment, flow_name=ctx.obj.flow.name
+    )
+    ctx.obj.event_logger.start()
```

Comment:
For my own curiosity - what additional events do we now capture by starting the logging here compared to before?

Reply:
Right now, nothing since we log all of two events. We are working on logging a lot more to get more insight on what is happening in the runtime. It's also more consistent with the monitor part.